src/geocat/viz/util.py

```diff
@@ -1347,7 +1353,7 @@ def plot_extrema_labels(da: xr.DataArray,
     clabel_points = proj.transform_points(
         transform, np.array([x[0] for x in label_locations]),
         np.array([x[1] for x in label_locations]))
-    transformed_locations = [(x[0], x[1]) for x in clabel_points]
```

Comment:
@kafitzgerald I changed this line to remove nans before the for loop so we wouldn't get that Error raised anymore, however I'm not sure if this is what we want to do. Perhaps raising the error down below was what we want so the user knows a certain point was excluded from the plot. I can change that error message to actually list which point was problematic? I'm not sure how to manipulate our example to just exclude this point in the first place in the cleanest way.

Reply:
Actually looking at it more the try except block isn't catching this nan and the except block is never called in this instance despite raising an error.